packages/pwa-kit-build/src/configs/webpack/config.js

```diff
@@ -309,7 +320,8 @@ const ssr = (() => {
                         path: buildDir,
                         filename: 'ssr.js',
                         libraryTarget: 'commonjs2'
-                    }
+                    },
+                    plugins: [analyzeBundle && getBundleAnalyzerPlugin('ssr')].filter(Boolean)
```

Comment:
Please make sure that we extend from existing plugins. And double check other similar lines.